### ISSUE TITLE
Enhancement: show scrollable list of additional strings

### ISSUE THREAD

**Issue by Kaligula0** [PhilippC/keepass2android#1413]
While editing an entry, and adding an additional string: it would be very handy if the app could show a list additional strings, so that man wouldn't have to remember the strings' names.

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
The additional fields have typing suggestions, but the entry tags labels "keywords" do not have. Do you mean this?

In desktop App you can quickly choose from between multiple existing entries but in mobile you have to type and type again.

**Comment by Kaligula0** [comment]
I agree, additional fields have typing suggestions, but only after typing AFAIR 3 chars. Some attributes have names I can't recall, so... 🤷🏻‍♂️ I can only type whole alphabet for the suggestions to remind me of it.

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
For the field name completion, it actually does show up suggestions after only entering the first 2 TWO characters.

-----Original Message-----

**Comment by Kaligula0** [comment]
That doesn't change anything, man still has to remember at least the beginning of the string's name.
